src/components/Modal/UIModal.js

```diff
@@ -33,15 +33,19 @@ const UiDataModal = ({
         serverPort: '',
         withExample: 'false',
         applicationType: 'gateway',
-        theme: 'default',
+        theme: '',
         ...CurrentNode,
     };
     const [UiData, setUiDataData] = useState(IntialState);
     const [duplicateApplicationNameError, setDuplicateApplicationNameError] = useState(false);
     const [portNumberError, setPortNumberError] = useState(false);
     const [clientFrameworkError, setClientFrameworkError] = useState(false);
+    const [applicationFrameworkError, setApplicationFrameworkError] = useState(false);
+    const [themeError, setThemeError] = useState(false);
     const isEmptyUiSubmit =
-        UiData.applicationName === '' || (UiData.applicationFramework === 'ui' && UiData.packageName === '') || UiData.serverPort === '';
+        UiData.applicationName === '' ||
+        // (UiData.applicationFramework === 'ui' && UiData.packageName === '') ||
```

Comment:
UiData.applicationName === '' || ....??
 
if there is not to be checked no need of or (||) operator here!
 
 remove the comment if not required.